### PR TITLE
Develop 1.0 - Base Template

### PR DESCRIPTION
**Pull request by akshay2905**
- Create a base template which would be inherited by other templates.
- Add Homepage skeleton.

### CODE REVIEW

File: assam_budget_board/templates/home.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            var target = $(this.hash);
+            target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
+            if (target.length) {
+              $('html, body').animate({
+                scrollTop: (target.offset().top - 54)
+              }, 2000, "easeInOutExpo");
+              return false;
+            }
+          }
+        });*/
```
Comment (curioswati) [inline assam_budget_board/templates/home.html]: should get rid of commented code, if it's not needed now. we can always have a look back through commit history if have to refer later.

File: assam_budget_board/templates/home.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                <i class="fa fa-bars"></i>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    {% show_menu 0 1 1 1 "menu/home_menu.html" %}
+                </ul>
+            </div>
+        </div>
+    </nav>
+{% endblock navbar %}
```
Comment (curioswati) [inline assam_budget_board/templates/home.html]: is the navbar going to be updated by any child template? if not then we could directly keep the code here without the block?

Comment (akshay2905) [reply assam_budget_board/templates/home.html]: Yes. The Navbar for Homepage and rest of the pages would be different thus, the block.

File: assam_budget_board/templates/home.html (hunk 1 above)
Comment (akshay2905) [reply assam_budget_board/templates/home.html]: Would eventually do. I am anticipating some changes, so once the content structure is a bit more organized and concrete, we would make sure all the commented code would be removed. Would keep this in mind for future commits. Thanks

**Comment by shreyaagrawal0809** [comment]
Using strict mode will restrict us from using delete function, will this create any hindrance in further development?
Rest of the code seems good to me

**Comment by shreyaagrawal0809** [comment]
Using strict mode will restrict us from using delete function, will this create any hindrance in further development?
Rest of the code seems good to me

**Comment by shreyaagrawal0809** [comment]
Using strict mode will restrict us from using delete function, will this create any hindrance in further development?
Rest of the code seems good to me

**Comment by shreyaagrawal0809** [comment]
Using strict mode will restrict us from using delete function, will this create any hindrance in further development?
Rest of the code seems good to me

**Comment by akshay2905** [comment]
I am not sure if we would need the delete function. But even if we do, we can remove the strict keyword from the js code of that specific snippet. Currently, the snippet in this PR is specifically written for the homepage.

**Comment by akshay2905** [comment]
I am not sure if we would need the delete function. But even if we do, we can remove the strict keyword from the js code of that specific snippet. Currently, the snippet in this PR is specifically written for the homepage.

**Pull request merged**